ideas/README.md

```diff
@@ -44,33 +58,33 @@ Pick up a book and read, here are a few that are invaluable
 * Practical Object Oriented Design in Ruby (Sandi Metz)
 * The Software Craftsman (Sandro Mancuso)
 
-## Watch
+### Watch
 
 Watch screencasts and other programming related videos
 
 * [Destroy All Software](https://www.destroyallsoftware.com/screencasts)
 * [Clean Coders](https://cleancoders.com/videos)
 * [Rails Casts](http://railscasts.com/)
```

Comment:
I'd mention it has quite a bit of deprecated videos, especially in free version.